localeval.js

```diff
@@ -1,6 +1,10 @@
 // Thaddée Tyl <thaddee.tyl@gmail.com>. License: CC-BY v3.
 "use strict";
 var node_js = typeof exports === 'object';
+if ("undefined" !== typeof process && process.browser) {
+    // webpack sets browser property on process global
+    node_js = false;
+}
```

Comment:
Fair point! We can break it down with another variable. The piece about webpack can be a `webpack` variable. Then `var node_js = typeof exports === 'object' && !webpack`.